### ISSUE TITLE
Add Anna's Collection

### ISSUE THREAD

**Issue by Alexoo8** [gruijter/com.gruijter.clusterlights#7]
Can you add Anna's Collection to the standards? E.g. Used by Tree Cluster. Thanks!

**Comment by gruijter** [comment]
I might if you donate a light. I need that to reverse engineer the protocol and test with Homey. Mind you, there is no guarantee this will succeed.

Let me know if you are willing to do that.

**Comment by Alexoo8** [comment]
Ik zie dat ik dit ook in het Nederlands kan doen.
Dat is mogelijk, maar ik kan geen goedkopere oplossing vinden dan degene hieronder.

https://www.intratuin.nl/miniverlichting-met-app-18-m-met-180-lampjes-classic-warm.html

**Comment by gruijter** [comment]
I like to keep it in English here in case other people want to read along.

The light is 23 euro plus 5 euro shipping (due to lock-down it needs to be ordered)

I can definitely give it a try. You can donate 28 euro here so I can order it:

www.paypal.me/gruijter

**Comment by Alexoo8** [comment]
OK, let's give it a try.

**Comment by gruijter** [comment]
Got the donation. Light is ordered!

**Comment by gruijter** [comment]
Hi Alex,

Bad news :(

I received the lights today and immediately started reverse engineering the bluetooth communication. Unfortunately I found out that the protocol is secured in an unknown way. It has to do with the network ID, but also with the original app keeping track of a command index, and I also suspect some type of encryption.

I googled if someone has hacked the protocol, but I cannot find anything.

Conlusion is: I am not able to add this into the Homey app, sorry.
